src/components/CityList.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import '../style/CityList.css';
+
+const CityList = ({ cities, getWeatherImage }) => (
+  <div className="city-list-container">
+    {cities.map((city) => (
+      <div key={city.id} className="city-box">
+        <img src={getWeatherImage(city.main.temp)} alt="Weather" className="weather-image" />
+        <Link to={`/details/${city.id}`} className="city-link">
+          {city.name}
+        </Link>
+        <p className="city-temperature">
+          {city.main.temp}
+          °C
+        </p>
+      </div>
+    ))}
+  </div>
+);
```

Comment:
- Hi, you did a nice job working on the design as requested by the previous reviewer, but you forgot to implement the arrows on the top right of each city. Without these arrows, it will be difficult for the user to Enter the city map to check the details.
- Your App:
![Capture](https://github.com/MussieTeka/the-weather-app/assets/63972670/9c9cf1a0-3be2-46ea-ad14-f541fc4026a0)
- Original:


![Capture](https://github.com/MussieTeka/the-weather-app/assets/63972670/a24401cb-477e-45da-a912-ab9e9bd48c9e)

Reply:
Thank you for your feedback @OLIPLICHE. I have now added and positioned an arrow on the top right corner of each city card.